ansible/README.md

```diff
@@ -87,12 +87,18 @@ ansible-playbook -i environments/<environment> couchdb.yml -e mode=initdb
 ansible-playbook -i environments/<environment> openwhisk.yml
 ```
 
-You need to run "initdb" on couch **every time** you deploy couchdb to initialize the db.
+You need to run `initdb` on couchdb **every time** you deploy couchdb to initialize the db.
+
+**Hint:** The `initdb` mode will ask for your confirmation to drop the database. To move on you need to enter `CTRL-C + C`. To abort you need to enter `CTRL-C + A`. You can skip this confirmation step by adding `-e prompt_user=false` to the command (only for `couchdb`, not for `cloudant`)
+
```

Comment:
The doc still mentions prompt bypassing above and below.

cloudant should be Cloudant

couchdb should be CouchDB